contrib/exporters/allinone/allinone.yml.default

```diff
@@ -14,8 +22,8 @@ pipeline:
       # - internal
       # - other
   transform:
-    type: sa
-    sa:
+    type: secadvisor
+    secadvisor:
       # exclude_started_flows: true
```

Comment:
I suggest moving `exclude_started_flows` outside secadvisor and to be directly under transform so that it can be used by other types of transforms, independent of the secadvisor transform (such as vpclogs).

Reply:
as it is it can be used by vpclogs if the transformer was registered as vpclogs then you would configure using:

```
pipeline:
  tranform:
    type: vpclogs
    vpclogs:
      exclide_started_flows: true
```